src/arithmetic/entity_funcs/entity_funcs.c

```diff
@@ -204,7 +204,15 @@ SIValue AR_PROPERTY(SIValue *argv, int argc, void *private_data) {
 
 	SIValue obj = argv[0];
 
-	if(SI_TYPE(obj) & SI_GRAPHENTITY) {
+	if(SI_TYPE(obj) == T_POINT)
```

Comment:
please move this if to be the last one as SI_GRAPHENTITY will be the most frequently used

Reply:
Ok. This change is done in commit [1cd2ed4](https://github.com/RedisGraph/RedisGraph/pull/2524/commits/1cd2ed4b41fcd64b160a7e55fcbef39c8379147a). I left the final else returning a SI_NullVal(), is that fine?